opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -191,7 +215,20 @@ class AquiferCarterTracy : public AquiferInterface<TypeTag>
         }
 
         this->pa0_ = this->aquct_data_.initial_pressure.value();
-        this->rhow_ = this->aquct_data_.waterDensity();
+        if(this->co2store_()) {
+             const auto press = this->aquct_data_.initial_pressure.value();
+
+             Scalar temp = FluidSystem::reservoirTemperature();
+             if (this->aquct_data_.initial_temperature.has_value())
+                 temp = this->aquct_data_.initial_temperature.value();
+
+             Scalar rs = 0.0; // no dissolved CO2
```

Comment:
The `rs` variable is unused.  Did you mean to pass this as an argument to `inverseFormationVolumeFactor()`?

Reply:
yes. thanks for noticing. I will update accordingly.